e2e/vue3-vite-e2e/tests/utils/async-commands.ts

```diff
@@ -0,0 +1,48 @@
+import { exec } from 'child_process';
```

Comment:
@abemscac I copied in the test utils directly here from `@nrwl/nx-plugin/testing` and updated them to allow parallel execution. As a result, all the test commands take an extra parameter `projectDir`.